### ISSUE TITLE
See if it is possible to ignore parsing struct fields as labels and causing errors when creating globules

### ISSUE THREAD

**Issue by mellonpizza** [Underrout/LunarHelper#36]
From a quick test module that defines a struct (I tend to use structs alot for organizing data arrangements, which have been frustrating from other warnings/errors I get in other tools from how asar treats them, so i acknowledge that this may be impossible of infeasible to address simply from how asar implements struct definitions), lunar helper causes an error by trying to define struct fields to a vaule which makes asar report there is an invalid label (because of the `struct.field` format) and throw an error. I have not looked into how asar implements structs in detail, and do not know if there is a way to disambiguate a genuine struct label and an actual improperly written label with a period in it, but I would consider it to make sense to treat them as more a define rather than a label in processing them if it is possible to disambiguate them.
![LunarHelper_Jb9fAjuJST](https://github.com/Underrout/LunarHelper/assets/22580852/e33f74f7-2d38-4b18-95a4-b67f7153fe57)
![image](https://github.com/Underrout/LunarHelper/assets/22580852/9398cc90-c38d-456a-9300-ae38b012e740)

**Comment by mellonpizza** [comment]
Update: it seems to be related to the 2 files being included and having the struct defined in both (one via incsrc), I technically do not need to have the second file be included though it is convenient to have next to the main module asm file; for now I will just rename it to something else. I am not sure if the remaining error is lunar helper's fault or not, so feel free to close the issue if this seems to not be anything on this program's end.

**Comment by Underrout** [comment]
I'm not sure I can reproduce the exact issue but I'm guessing it's something to do with Lunar Helper putting stuff like this in the generated globule file when you've got a struct:
```
test_Test.a = $008000
!test_Test.a = $008000
test_Test.b = $008002
!test_Test.b = $008002```

I don't use structs, so I've never ran into this personally, kinda funny that asar apparently reports these as labels when you tell it to return all labels in an assembled file. I'm guessing you will probably get an error when trying to include the generated file due to the defines/labels looking like the above snippet, though idk why it'd already show errors when assembling the globule itself.

Should be pretty easy to just check for a dot in the label name and just not include it, I'm already doing something similar for relative labels (+, -, ++, etc.). I'm assuming that would probably fix the problem.

**Comment by Underrout** [comment]
I believe this should now be fixed in the newest release (v4.0.6), which simply discards labels like these.
